helpers/postgres.rb

```diff
@@ -130,6 +130,50 @@ def generate_postgres_options(flavor: nil, location: nil)
     options.serialize
   end
 
+  PostgresLocation = Struct.new(:location, :available_postgres_versions, :available_vm_families)
+  def vm_families_for_project(project)
+    locations = Location.visible_or_for_project(@project.id, @project.feature_flags["visible_locations"])
+    locations.map do |location|
+      PostgresLocation.new(
+        location: location,
```

Comment:
Hash value omission:

```suggestion
        location:,
```

Can be applied to other cases in this PR.